src/agent/onefuzz-agent/src/main.rs

```diff
@@ -39,6 +39,12 @@ fn main() -> Result<()> {
                 .short("c")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("setup_dir")
+                .long("setup_dir")
+                .short("sd")
```

Comment:
FYI, only the `s` will get used, and `-sd` will not work as expected. See the `NOTE` section [here](https://docs.rs/clap/2.33.3/clap/struct.Arg.html#method.short).
> only the first non `-` character will be used as the short version